requirements.txt

```diff
@@ -10,3 +10,4 @@ warc3-wet>=0.2.3
 warc3-wet-clueweb09>=0.2.5
 zlib-state>=0.1.3
 ijson>=3.1.3
+pyautocorpus>=0.1.0
```

Comment:
This is a task for me, but we'll need to solve the Windows problem with this package before we can push out a release of this. Otherwise, pip install will not work for Windows users.